### ISSUE TITLE
Recommended way for running Prerender is Docker

### ISSUE THREAD

**Issue by AlmogCohen** [prerender/prerender#652]
Hi there! I appreciate this awesome project and the fact you made it under the MIT license! (We are actually paying for pretender.io)

I was wondering if the project has a best-practice recommendation for running pretender within docker. There are a few non-official repos out there but I'm looking for something more official, more supported.

There are repos like this one https://github.com/Zenika/alpine-chrome that can handle most of the heavy-lifting of the headless chrome setup. Any recommendations?

**Comment by divyenduz** [comment]
I am running it with this https://github.com/divyenduz/prerender.zoid.in for a while now and I have had no issues so far :)

**Comment by ekifox** [comment]
@divyenduz can you please add your image to docker hub?

**Comment by viperfx** [comment]
@divyenduz do you use nginx middleware proxy? I am finding that my URLs are redirecting to the prerender server rather than serving the original content

**Comment by divyenduz** [comment]
I was using traefik or something but yes equivalent of a proxy.

**Comment by viperfx** [comment]
@divyenduz I see - are you still actively using this? and did you have the redirect issue mentioned in #699

**Comment by divyenduz** [comment]
Nope, this is no longer in use but was working as recently as last week when I shut the service down.

**Comment by viperfx** [comment]
@divyenduz Ah got it! You are using an alternative cloud option? or no longer need pre rendering? I only use this to get social cards working :)

**Comment by divyenduz** [comment]
Nah, it was a side project that I archived.

**Comment by Norlandz** [comment]
prerender-alpine/server.js at master · tvanro/prerender-alpine
https://github.com/tvanro/prerender-alpine/blob/master/server.js
- so far the decent good one

prerender-docker/server.js at master · ITachiLab/prerender-docker
https://github.com/ITachiLab/prerender-docker/blob/master/server.js
- too old, not working, but just a reference
